test/cpp/qps/client_async.cc

```diff
@@ -949,10 +949,182 @@ std::unique_ptr<Client> CreateAsyncClient(const ClientConfig& config) {
       return nullptr;
   }
 }
+
 std::unique_ptr<Client> CreateGenericAsyncStreamingClient(
     const ClientConfig& args) {
   return std::unique_ptr<Client>(new GenericAsyncStreamingClient(args));
 }
 
+struct CallbackClientRpcData {
+  CallbackClientRpcData()
```

Comment:
I was avoiding renaming it to context as we have a seperate ClientContext structure and this particular structure doesn't embed it like the Async test, hence this name to avoid any such assumption. I don't feel strongly about it though and will rename it if this justification does not seem valid.